src/getHumanAge.test.js

```diff
@@ -7,4 +7,37 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return 0, if animalAge less than first', () => {
+    const result = getHumanAge(14, 14);
+
+    expect(result).toEqual([0, 0]);
+  });
+
+  test(`should return 1,
```

Comment:
The test description is unclear. It should specify that `catAge` and `dogAge` are between 15 and 24, which corresponds to 1 human year according to the task description.